File: Esquenta/Entities/EntradaProduto.cs
using Esquenta.Entities.Interfaces;
using System;

namespace Esquenta.Entities
{
    public class EntradaProduto : IBaseEntity
    {
        public virtual int Id { get; protected set; }
        public virtual Produto Produto { get; set; }
        public virtual int Quantidade { get; set; }
        public virtual decimal Valor { get; set; }
        public virtual DateTime DataEntrada { get; set; }
    }
}

File: V2-Node/esquenta.v2/server/models/entradaproduto.js
'use strict'
module.exports = (sequelize, DataTypes) => {
  var entradaProduto = sequelize.define('entradaProduto', {
    quantidade: DataTypes.INTEGER,
    valor: DataTypes.DECIMAL,
    dataEntrada: DataTypes.DATE
  }, {})
  entradaProduto.associate = function (models) {
    // associations can be defined here
  }
  return entradaProduto
}
